_includes/components/department-budgets/DeptSectionHead/styles.scss

```diff
@@ -0,0 +1,21 @@
+
+.DeptSectionHead {
+  font-size: 2rem;
```

Comment:
Is there existing H-tag font sizing we can rely on here, and just add colour and padding? That would help keep the font sizing consistent.

Reply:
Right. However we don't have a typographic scale/system underlying the design. So binding styling to semantic tags will create a lot of problems for us. The best approach might be to reset all H-tags globally and then style them individually.